tracks/ruby/exercises/acronym/mentoring.md

```diff
@@ -21,14 +21,25 @@ with variations for the regex (see below).
   
 In all these cases: it's recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, give them also the RegEx (`/\b[a-zA-Z]/`). (At this point in the track, it's more about experiencing the strengths of Ruby's built-in methods than making them find them. There's ample opportunity for research later on in the track.)   
 
-## Talking points
-* If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
+### Talking points
+* `Class vs Module`: If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
 * Because of the regex needed for `split`, `scan` is in this exercise (marginally) faster.
 * It can be hard to catch all the possible delimiters. 
 * `upcasing` the whole string is (10% ?) slower than `upcasing` the acronym only. 
 * Store the regex in a constant or a variable, mostly so it can be named. 
+* `join("")`: The `("")` after `join` can be omitted, because it's the default for `join`.
 
-## Passing Regular Expressions
+
+### Mentoring strategy
+* When they start with `split`, with or without `tr` or `gsub`, and `map` to `upcase`: start with pointing them to `scan`, name what it accomplished afterwards:
+> 1) (first comment)    
+  Using split works, but it forces you to focus on the delimiters (space, -). Ruby has `String#scan`. With `scan` and a regex (Regular Expression), you can catch the parts you need. For instance, `scan` with the regex `/\b[a-zA-Z]/` will catch 'word boundaries' (space, - and more) followed by a word character.
+I use "www.rubular.com" whenever I want to check a regex. 
+> 2) (after they implemented scan)
```

Comment:
Are we considering indenting by 4 spaces those things that we want to indicate is copy/paste material?

Reply:
The first one is indented with 2 not 4 spaces. It's to start them on the next line. I adjusted the second one accordingly! Thnx :-)

Reply:
I was thinking more like this:

1. first comment
>  Here are some copy paste comments.
2. second comment
> Here are the second copy paste comments.